core/node/main/config.hpp

```diff
@@ -34,8 +34,10 @@ namespace fc::node {
     // drand config
     std::vector<std::string> drand_servers;
     BlsPublicKey drand_bls_pubkey;
+    /** Drand genesis time in seconds */
     int64_t drand_genesis = 0;
-    int64_t drand_period = 0;
+    /** Drand round time in seconds */
+    int64_t drand_period = 1;
```

Comment:
0 is invalid period. Lotus default is 1 (see https://github.com/filecoin-project/lotus/blob/master/chain/gen/gen.go#L250)

Reply:
Decided to keep 0 as not set.